lisp/php-mode.el

```diff
@@ -1287,7 +1287,7 @@ for \\[find-tag] (which see)."
         "never" "never-return" "never-returns" "no-return" "non-empty-array"
         "non-empty-list" "non-empty-string" "non-falsy-string"
         "numeric" "numeric-string" "positive-int" "scalar"
-        "trait-string" "truthy-string"))
+        "trait-string" "truthy-string" "self-out" "this-out"))
```

Comment:
`self-out` and `this-out` are not type. Please add to `php-phpdoc-type-tags`.

Reply:
I fixed it now!